plugins/BEdita/Core/src/ORM/QueryFilterTrait.php

```diff
@@ -85,16 +85,29 @@ public function fieldsFilter(Query $query, array $options)
                 }
 
                 $in = [];
+                $nin = [];
                 foreach ($conditions as $operator => $value) {
                     if (is_numeric($operator)) {
                         $in[] = $value;
                         continue;
                     }
-                    $exp = $this->operatorExpression($exp, $operator, $field, $value);
+
+                    switch ($operator) {
+                        case 'nin':
+                            $nin = is_string($value) ? explode(',', $value) : (array)$value;
+                            break;
```

Comment:
You should be able to put the same logic inside `operatorExpression()`. Doing so you would leave this method untouched and simply add a few lines there.

```php
switch ($operator) {
    case 'eq':
    // ... 

    case 'nin':
    case 'notin':  // Optionally add a few aliases
        $exp = $exp->notIn($field, is_string($value) ? explode(',', $value) : (array)$value);

    // ...
```